### DISCUSSION TITLE
Mediamtx on kubernetes

### DISCUSSION THREAD

**Discussion by Romain-Fromantin** [bluenviron/mediamtx#3086] [Questions]
### Question

I launched a deployment of Server MediaMTX. To operate Whip I added Stunner as Server Turn.I added the turn server in the mediamtx conf.
Should I add the turn server to the client (OBS We can't do it )?
If I redirect the first query /mystream/whip to the server I want I can choose on which server will be launched the stream ? (He will manage to connect to the client with the turn server)?

**Comment by aler9** [comment]
Hello, native scalability is a feature that is currently missing, you can follow the development here: #998

However, i think that if you are using HTTP-based protocols only (like WebRTC and HLS), you can route requests from the same user to the same replica by using Sticky Sessions, that should be supported natively by most load balancers (and even Kubernetes ingresses).

[a comment by an automated account is not shown]
